packages/sdk-node/src/__tests__/__snapshots__/exports.test.ts.snap

```diff
@@ -36,6 +36,7 @@ exports[`module exports should not change unless explicitly set 1`] = `
   "createClerkClient",
   "createClerkExpressRequireAuth",
   "createClerkExpressWithAuth",
+  "createIsomorphicRequest",
```

Comment:
❓ @nikosdouvlis shouldn't we omit the `createIsomorphicRequest` from the `sdk-node` (and nextjs) package?

Reply:
We decided to let it be reexported.